lib/src/entrypoint.dart

```diff
@@ -302,6 +302,10 @@ See $workspacesDocUrl for more information.''',
     p.normalize(p.join(workspaceRoot.dir, '.dart_tool', 'package_config.json')),
   );
 
+  late final String packageGraphPath = p.relative(
+    p.normalize(p.join(workspaceRoot.dir, '.dart_tool', 'package_graph.json')),
```

Comment:
I would wish for the word 'dependency' to be there somehow, but not blocking on it.